beginner_source/inference_on_aws_graviton_tutorial.rst

```diff
@@ -0,0 +1,343 @@
+           aten::addmm        87.81%        3.613s        91.90%        3.781s      12.604ms           300  
+       aten::clamp_min         7.18%     295.437ms         7.18%     295.437ms       1.477ms           200  
+           aten::copy_         4.07%     167.516ms         4.07%     167.516ms     558.387us           300  
+     mymodel_inference         0.67%      27.708ms       100.00%        4.115s        4.115s             1  
+          aten::linear         0.06%       2.499ms        92.06%        3.788s      12.627ms           300  
+               aten::t         0.05%       1.982ms         0.11%       4.385ms      14.617us           300  
+            aten::relu         0.05%       1.932ms         7.23%     297.369ms       1.487ms           200  
+======================  ============  ============  ============  ============  =============  ============= 
+
+``Self CPU time total: 4.115s``
+
+
+and following is the profiler output with the openBLAS backend:
+
+``$ export TORCH_MKLDNN_MATMUL_MIN_DIM=64``
+
+.. table:: output with the openBLAS backend
+
+======================  =============  ============  ============  ============  =============  ============   
+                Name     Self CPU %      Self CPU     CPU total %   CPU total    CPU time avg    # of Calls
+======================  =============  ============  ============  ============  =============  ============
+           aten::addmm        92.66%        1.179s        95.23%        1.211s       4.038ms           300
+       aten::clamp_min         2.83%      36.060ms         2.83%      36.060ms     180.300us           200
+           aten::copy_         2.52%      32.013ms         2.52%      32.013ms     106.710us           300
+     mymodel_inference         1.38%      17.521ms       100.00%        1.272s        1.272s             1
+          aten::linear         0.14%       1.750ms        95.60%        1.216s       4.054ms           300
+               aten::t         0.12%       1.475ms         0.24%       3.033ms      10.110us           300
+            aten::relu         0.10%       1.285ms         2.94%      37.345ms     186.725us           200
+======================  =============  ============  ============  ============  =============  ============
+
+``Self CPU time total: 1.272s``
+
+
+Here we observed ``3.2x (1.272s vs 4.115s)`` performance improvement by tuning the backend thresholds appropriately. We are working on making the backend selection automatic, with robust heuristics, for the future releases.
+
+
+4. Optimize memory allocation overhead with Linux Transparent huge pages
+-------------------------------------------------------------------------
+
+We also observed that for these larger networks, tensor memory allocations take significant portion of the inference latency. This can be optimized by enabling Linux transparent huge pages allocations from PyTorch C10 memory allocator. Set the following environment variable to enable it.
```

Comment:
```suggestion
We also observed that for these larger networks, tensor memory allocations take significant portion of the inference latency. This can be optimized by enabling Linux transparent huge pages allocations from PyTorch C10 memory allocator. Set the following environment variable to enable it:
```